### PR TITLE
Adding darkwolf theme, submitted as an attachment to pull request #51

### PR DESCRIPTION
**Pull request by ibennetch**
See https://github.com/phpmyadmin/themes/pull/51 and
https://github.com/phpmyadmin/themes/files/5541353/darkwolf.zip

### CODE REVIEW

File: darkwolf/theme.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "darwolf",
```
Comment (ibennetch) [inline darkwolf/theme.json:2]: Probably this is mean to be "darkwolf"

File: darkwolf/theme.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    "version": "5.0",
+    "description": "Default phpMyAdmin theme",
```
Comment (ibennetch) [inline darkwolf/theme.json:4]: I hope we can come up with a slightly better description

File: darkwolf/theme.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    "author": "phpMyAdmin developers",
```
Comment (ibennetch) [inline darkwolf/theme.json:5]: I've asked the author in https://github.com/phpmyadmin/themes/pull/51#issuecomment-727464939 if there's a better name or alias to use here. I don't like to claim authorship of something we didn't make :)

**Review by ibennetch — COMMENTED** [review]
Probably the .bak files are not needed and should be removed.

File: darkwolf/theme.json (hunk 2 above)
Comment (SteppenWolf68) [reply darkwolf/theme.json]: > I hope we can come up with a slightly better description

Hi @ibennetch. Thanks for your comments i create a new issue and upload the files with the fix json

**Comment by ibennetch** [comment]
Thanks for fixing those things. I quite like the theme, you've done nice
work.

**Comment by SteppenWolf68** [comment]
> Thanks for fixing those things. I quite like the theme, you've done nice work.

I am glad that you liked my work.
I hope it will be useful and comfortable when working.

**Pull request closed without merging**